caasp-velero-restic-restore-helper-image/caasp-velero-restic-restore-helper-image.kiwi

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- OBS-AddTag: caasp/v4/velero-restic-restore-helper:%%PKG_VERSION%%-rev<VERSION> caasp/v4/velero-restic-restore-helper:%%PKG_VERSION%%-rev<VERSION>-build<RELEASE> -->
+
+<image schemaversion="6.9" name="caasp-velero-restic-restore-helper-image" xmlns:suse_label_helper="com.suse.label_helper">
+  <description type="system">
+    <author>SUSE Containers Team</author>
+    <contact>containers@suse.com</contact>
+    <specification>velero-restic-restore-helper running on a SLES15 SP1 container guest</specification>
+  </description>
+  <preferences>
+    <type
+      image="docker"
+      derived_from="obsrepositories:/suse/sle15#15.1">
+      <containerconfig
+        name="caasp/v4/velero-restic-restore-helper"
+        tag="%%PKG_VERSION%%"
+        maintainer="SUSE Containers Team &lt;containers@suse.com&gt;"
+        user="nobody">
+        <entrypoint execute="/velero-restic-restore-helper"/>
+        <subcommand clear="true"/>
+        <labels>
+          <suse_label_helper:add_prefix prefix="com.suse.caasp.v4">
+            <label name="org.opencontainers.image.description" value="velero-restic-restore-helper running on a SLES15 SP1 container guest"/>
+            <label name="org.opencontainers.image.title" value="CaaSP velero-restic-restore-helper container"/>
+            <label name="org.opencontainers.image.created" value="%BUILDTIME%"/>
+            <label name="org.opencontainers.image.vendor" value="SUSE LLC"/>
+            <label name="org.opencontainers.image.url" value="https://www.suse.com/products/caas-platform/"/>
+            <label name="org.opencontainers.image.version" value="%%PKG_VERSION%%"/>
+            <label name="org.openbuildservice.disturl" value="%DISTURL%"/>
+            <label name="com.suse.reference" value="registry.suse.com/caasp/v4/velero-restic-restore-helper:%%PKG_VERSION%%"/>
+          </suse_label_helper:add_prefix>
+        </labels>
+      </containerconfig>
+    </type>
+    <version>1</version>
+    <packagemanager>zypper</packagemanager>
+    <rpm-excludedocs>true</rpm-excludedocs>
+  </preferences>
+  <repository>
+    <source path="obsrepositories:/"/>
+  </repository>
+  <packages type="image">
+    <package name="velero-velero-restic-restore-helper"/>
```

Comment:
velero-velero-restic-restore-helper? Is the duplicated velero intended?

Reply:
LGTM, except this one.

Reply:
It is intended.  When we build as subpackage using spec. It will have <PAKCAGE_NAME>-<SUB_PACKAGE_NAME>.  In this case, PACAKGE_NAME is `velero`, and SUB_PACKAGE_NAME is `velero-restic-restore-helper`.

Reply:
Thanks for clarifying.

Reply:
Nah, that's *really* strange. Please see https://build.suse.de/request/show/208735 for a cleaner approach.

Reply:
Just created https://github.com/SUSE/caasp-container-images/pull/49 to fix this.